src/command_modules/azure-cli-role/azure/cli/command_modules/role/commands.py

```diff
@@ -105,12 +105,29 @@ def get_graph_client_groups(_):
 cli_command(__name__, 'ad user create', 'azure.cli.command_modules.role.custom#create_user',
             get_graph_client_users)
 
+cli_command(__name__, 'ad group create',
+            'azure.graphrbac.operations.groups_operations#GroupsOperations.create',
```

Comment:
the string `azure.graphrbac.operations.groups_operations` is repeatedly used. i can be extract to a variable.